flow.js

```diff
@@ -96,6 +97,9 @@ class Flow {
       if(nextFlowStep == 'repair-description' && repairProblemChanged){
         delete state.data['repairProblemBestDescription']
       }
+      if( state.data.communal == 'yes' && state.step =='address') {
```

Comment:
Use `RepairType.isCommunalRepairType` helper function, 
```suggestion
      if(isCommunalRepairType(state.data) == 'yes' && state.step =='address') {
```